Assets/Scripts/Controllers/ActorController.cs

```diff
@@ -227,6 +227,8 @@ protected virtual void Die()
         GameObject enemyDropGameObject = (GameObject) Instantiate(enemyDrop,
             transform.position,
             Quaternion.identity);
+        Transform spawnableParent = GameObject.Find("Spawnables").transform;
```

Comment:
This same code shows up several times (playerController, Spawners). Could we just put it in Start() of this class and make it accessible through a spawnablesParent variable?